packages/content-management/table_list_view_table/tsconfig.json

```diff
@@ -26,6 +26,8 @@
     "@kbn/shared-ux-link-redirect-app",
     "@kbn/test-jest-helpers",
     "@kbn/content-management-table-list-view-common",
+    "@kbn/core",
```

Comment:
probably nit: I prefer to depend on specific packages types in packages instead of depending on the whole core